backend/db/migrations/V0.23.1__CE-618.sql

```diff
@@ -0,0 +1,91 @@
+-- 
+-- Create new tables for action_taken
+-- 
+CREATE TABLE
+  public.action_taken (
+    action_taken_guid uuid NOT NULL DEFAULT uuid_generate_v4 (),
+    complaint_identifier VARCHAR(20) NULL,
+    complaint_update_guid uuid NULL,
```

Comment:
Need a check constraint the either complaint_identifier or complaint_update_guid is provided